App/components/ExternalLink/ExternalLink.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactNode } from 'react';
+import { Text, GestureResponderEvent } from 'react-native';
+
+import { link } from '../../util/theme';
+
+interface ExlinkProps {
+	dest: (event: GestureResponderEvent) => void;
+	children: ReactNode
+}
+
+export function Exlink(props: ExlinkProps): React.ReactElement {
+	const { children, dest } = props
+	return <Text onPress={dest} style={link}>{children}</Text>
+}
```

Comment:
I think this will save us from a lot of trouble of retyping. I hope you would accept this proposal. Also, I'm considering between 2 names: `Exlink` or `Link`. Which do you think is better?